applications/rag/frontend/main.tf

```diff
@@ -30,6 +34,43 @@ locals {
     instance_connection_name = format("%s:%s:%s", var.project_id, var.region, "pgvector-instance")
 }
 
+# IAP Section: Enabled the IAP service
+resource "google_project_service" "project_service" {
+  project = var.project_id
+  service = "iap.googleapis.com"
+
+  disable_dependent_services = false
+  disable_on_destroy         = false
+}
+
+# IAP Section: Creates the OAuth client used in IAP
+resource "google_iap_client" "iap_oauth_client" {
+  count        = var.client_id == "" ? 1 : 0
+  display_name = "Frontend-Client"
+  brand        = var.brand == "" ? "projects/${data.google_project.project.number}/brands/${data.google_project.project.number}" : var.brand
+}
+
+# IAP Section: Creates the GKE components
+module "iap_auth" {
+  source = "../../../modules/iap"
```

Comment:
can we make this a fully qualified path? see egs in https://github.com/GoogleCloudPlatform/ai-on-gke/pull/228/files